src/components/PreviewOverlay.tsx

```diff
@@ -137,43 +111,15 @@ function PreviewOverlay({ actionList }: PreviewOverlayProps): JSX.Element {
                     <ActionTabSelectedListItem>
```

Comment:
all these components are useless in the current context so they can be removed and instead the call to trigger should be simplified